Add callout to ZK page (#7067)
ethereum/ethereum-org-website: src/content/developers/docs/scaling/zk-rollups/index.md

Looking for a more beginner-friendly resource? See our [introduction to layer 2](/layer-2/).

> You should also be familiar with ZKP technology. See our [primer on zero-knowledge proofs](/zero-knowledge-proofs/).

Reviewer: ```suggestion
You should also be familiar with zero knowledge as a concept. See our [introduction to zero-knowledge proofs](/zero-knowledge-proofs/).
```

Reviewer: Maybe not as a concept... brain has switched off

Reviewer 2: ```suggestion
You should also be familiar with zero-knowledge proofs. See our [introduction to zero-knowledge proofs](/zero-knowledge-proofs/).
```

Not a strong opinion. In favor of a mixture of the two, but would prefer no acronym I think.
